### PR TITLE
Update inputs in condition section

### PR DESCRIPTION
**Pull request by andrzejewsky**
issue: https://github.com/piwik/plugin-CustomAlerts/issues/22

### CODE REVIEW

File: stylesheets/alerts.less [hunk 1]
```diff
@@ -18,7 +18,8 @@
   }
 
   .entityListContainer, .entityAddContainer {
-    width: 800px;
```
Comment (adaqus) [inline stylesheets/alerts.less]: Good idea.

**Comment by adaqus** [comment]
@andrzejewsky :+1:

**Comment by andrzejewsky** [comment]
@mattab Could you take a look? It should solve issue: https://github.com/piwik/plugin-CustomAlerts/issues/22

**Comment by tsteur** [comment]
It does now look like 
image in Chrome 47 on MacOS. Is it intended to have the last input field in a new line?

**Comment by andrzejewsky** [comment]
@tsteur I made some changes and here is my results (I have the same OS and Browser):

zrzut ekranu 2016-01-11 o 16 22 56

the same it looks like  in other browsers.

**Comment by tsteur** [comment]
Yes it works now for some reason :) Thx :)

Just one more thing. Can you maybe add a `min-width:800px` otherwise the initial table looks a bit small, especially on a big screen. 

image

**Comment by andrzejewsky** [comment]
@tsteur done.

**Comment by tsteur** [comment]
Thx, looks great!

**Comment by hpvd** [comment]
would be good to release a new version in plugin manager...

**Comment by tsteur** [comment]
Will do now

**Comment by hpvd** [comment]
thanks!
sorry, had to open a complementing one: https://github.com/piwik/plugin-CustomAlerts/issues/28

**Pull request merged**